rvv-intrinsic-generator/rvv_intrinsic_gen/vector_crypto_inst.py

```diff
@@ -0,0 +1,210 @@
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector AES Encryption",
+      "",  # FIXME: We probably have a separate document for vector-crypto
+      ["vaesef", "vaesem"],
+      UITYPE,
+      [32],
+      LMULS,
+      decorators.has_no_masking_policy)
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector AES Decryption",
+      "",  # FIXME: We probably have a separate document for vector-crypto
+      ["vaesdf", "vaesdm"],
+      UITYPE,
+      [32],
+      LMULS,
+      decorators.has_no_masking_policy)
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector AES-128 Forward KeySchedule generation",
+      "",  # FIXME: We probably have a separate document for vector-crypto
+      ["vaeskf1", "vaeskf2"],
+      UITYPE,
+      [32],
+      LMULS,
+      decorators.has_no_masking_policy)
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector AES round zero",
+      "",  # FIXME: We probably have a separate document for vector-crypto
+      ["vaesz"],
+      UITYPE,
+      [32],
+      LMULS,
+      decorators.has_no_masking_policy)
+
+  ####################################################################
+
+  g.start_group("Zvknh - NIST Suite: Vector SHA-2 Secure Hash")
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector SHA-2 message schedule",
+      "",  # FIXME: We probably have a separate document for vector-crypto
+      ["vsha2ms"],
+      UITYPE,
+      [32, 64],
+      LMULS,
+      decorators.has_no_masking_policy)
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector SHA-2 two rounds of compression",
+      "",  # FIXME: We probably have a separate document for vector-crypto
+      ["vsha2ch", "vsha2cl"],
+      UITYPE,
+      [32, 64],
+      LMULS,
+      decorators.has_no_masking_policy)
+
+  ####################################################################
+
+  g.start_group("Zvksed - ShangMi Suite: SM4 Block Cipher")
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector SM4 KeyExpansion",
+      "",  # FIXME: We probably have a separate document for vector-crypto
+      ["vsm4k"],
+      UITYPE,
+      [32],
+      LMULS,
+      decorators.has_no_masking_policy)
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector SM4 Rounds",
+      "",  # FIXME: We probably have a separate document for vector-crypto
+      ["vsm4r"],
+      UITYPE,
+      [32],
+      LMULS,
+      decorators.has_no_masking_policy)
+
+  ####################################################################
+
+  g.start_group("Zvksh - ShangMi Suite: SM3 Secure Hash")
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector SM3 Message Expansion",
+      "",  # FIXME: We probably have a separate document for vector-crypto
+      ["vsm3me"],
+      UITYPE,
+      [32],
+      LMULS,
+      decorators.has_no_masking_policy)
+
+  g.function_group(
+      vector_crypto_template,
+      "Vector SM3 Message Expansion",
```

Comment:
It should be `Vector SM3 Compression` 

https://github.com/riscv/riscv-crypto/blob/main/doc/vector/insns/vsm3c.adoc